app/views/traces/_trace.html.erb

```diff
@@ -1,35 +1,44 @@
-<tr>
-  <td>
-    <% if Settings.status != "gpx_offline" %>
-      <% if trace.inserted %>
-        <a href="<%= url_for :controller => "traces", :action => "show", :id => trace.id, :display_name => trace.user.display_name %>"><img src="<%= url_for :controller => "traces", :action => "icon", :id => trace.id, :display_name => trace.user.display_name %>" border="0" alt="" /></a>
-      <% else %>
-        <span class="text-danger"><%= t ".pending" %></span>
-      <% end %>
+<li class="media mb-1 p-2 <%= cycle("", " bg-light") %>">
+  <% if Settings.status != "gpx_offline" %>
+    <% if trace.inserted %>
+      <a href="<%= url_for :controller => "traces", :action => "show", :id => trace.id, :display_name => trace.user.display_name %>">
+        <img src="<%= url_for :controller => "traces", :action => "icon", :id => trace.id, :display_name => trace.user.display_name %>" alt="" class="img-thumbnail mr-3" />
+      </a>
+    <% else %>
+      <div style="width: 60px; height: 60px" class="img-thumbnail bg-white mr-3"></div>
+    <% end %>
+  <% end %>
+
+  <div class="media-body">
```

Comment:
The media component has been removed from bootstrap 5. See https://github.com/twbs/bootstrap/pull/28265. So to avoid having to refactor this soon, we should avoid using it now.